tests/syscalls/syscalls_test.go

```diff
@@ -54,6 +57,73 @@ var _ = Describe("Syscalls", func() {
 		KspDeleteAll()
 	})
 
+	Describe("Annotation", func() {
```

Comment:
Syscall test suite is meant for testing syscalls related policies and events. Can we put this in a different test suite. Maybe ksp?
Also, can we add a simple check to see if policy enforcement works for pre-existing pods? It would be nice to have this for edge cases, we currently don't have any such test in ksp.